docs/install/iso-install.md

```diff
@@ -68,9 +98,16 @@ In some cases, if you are using an older graphics card/monitor, you may encounte
 
 ![invalid-dimensions.png](/img/v1.2/install/invalid-dimensions.png)
 
-This is a known issue we are working on, and will be fixed in future releases. Here is a workaround for this issue:
+This is a known issue we are working on, and will be fixed in future releases. You can try to use another GRUB entry to force it to use the resolution of `1024x768` when booting up.
+
+![force-resolution.png](/img/v1.2/install/force-resolution.png)
+
+If you are using a version earlier than v1.1.1, please try the following workaround:
+
 1. Boot up with the ISO, and press `E` to edit the first menu entry:
-   ![grub-menu.png](/img/v1.2/install/grub-menu.png)
+   ![grub-menu.png](/img/v1.1/install/grub-menu.png)
+
 2. Append `vga=792` to the line started with `$linux`:
-   ![edit-menu-entry.png](/img/v1.2/install/edit-menu-entry.png)
+   ![edit-menu-entry.png](/img/v1.1/install/edit-menu-entry.png)
```

Comment:
This is a v1.2 doc and it should reference images from `/img/v1.2`. Please double-check if the current images are fine, or should we copy the img to `/img/v1.2` and reference them from there instead

Reply:
It's a typo LOL 🤣